vestapol/web_resources/json_resource.py

```diff
@@ -11,7 +12,7 @@ class JSONResource(base_resource.BaseResource):
     response_filename = "data.json"
 
     def __init__(
-        self, name, base_url, endpoint, version, query_params=None, request_headers=None
+        self, name, base_url, endpoint, version, query_params=None, request_headers=None, requested_at=DateTime.utcnow().replace(microsecond=0)
```

Comment:
Do we need a `requested_at` parameter here? It's not used elsewhere in the class